static/src/js/components/CollapsePanel/__tests__/CollapsePanel.test.jsx

```diff
@@ -28,8 +28,8 @@ const setup = () => {
 
 describe('CollapsePanel component', () => {
   test('renders itself correctly', () => {
-    const { container } = setup()
-    expect(container.firstChild).toHaveClass('collapse-panel test-wrap-class')
+    setup()
```

Comment:
If no other tests are using the container, don't set it on line 19 and return it on line 22